std/root.ln

```diff
@@ -2411,7 +2411,7 @@ export fn newHashMap(firstKey: Hashable, firstVal: any): HashMap<Hashable, any>
   return hm.set(firstKey, firstVal);
 }
 
-export fn toHashMap(kva: Array<KeyVal<Hashable, any>>) {
+export fn toHashMap(kva: Array<KeyVal<Hashable, any>>): HashMap<Hashable, any> {
```

Comment:
The compiler at one point was complaining that the output type of this was `void`. I don't know how the type inference failed there, but I decided to just add it in if it was going to fail in edge cases.